requirements/app.in

```diff
@@ -46,6 +46,7 @@ Flask-Migrate
 Flask-WTF
 Flask-Mail
 # <5.2: https://github.com/Parallels/rq-dashboard/issues/417 as 5.2 requires Flask 2.3
+# Flask-Security-Too depends requires flask 2.3
```

Comment:
This line does not make sense grammatically.
Also, the line above also mentions Flask 2.3.
I guess we can delete them both as the < constraint is not here anymore?

Reply:
removed this line, i think this was in there by accident